cmd/dump.go

```diff
@@ -151,7 +155,12 @@ func runDump(ctx *cli.Context) error {
 		}
 	}
 
-	if com.IsExist(setting.LogRootPath) {
+	// Doesn't check if LogRootPath exists before processing --skip-log intentionally,
+	// ensuring that it's clear the dump is skipped whether the directory's initialized
+	// yet or not.
+	if ctx.IsSet("skip-log") {
```

Comment:
And is true?

Reply:
It wasn't clear to me if this was needed based on how skip-repository is handled just a bit earlier, also only checking if it's set